e2e/browser/package.json

```diff
@@ -4,9 +4,13 @@
   "description": "",
   "main": "index.ts",
   "scripts": {
-    "test": "npx testcafe all ess.test.ts"
+    "test": "testcafe firefox:headless,chrome:headless testSuiteRunner.test.ts"
```

Comment:
Just curious - why replace `all`? Is this just 'cos only firefox and chrome can run headlessly? And is the change to headless just for the speed improvement (which I'd totally buy into), but it seems a bit dangerous to leave our Edge and Safari, no...?

Reply:
We need to run it headless in CI as there's no display server there. I'd have preferred to do `all:headless`, but that's not possible yet: https://github.com/DevExpress/testcafe/issues/5641

(I'm not sure whether Safari can do headless. Both it and Edge are not available in every OS, so we'd have to modify our CI to run in different OS's and then adjust the list based on the OS. Which is a good idea, but for a future PR - especially if that issue gets implemented before that time.)

Reply:
@Vinnl - I see, very interesting - cheers (great job creating the TestCafe issue too!).